static/js/components/ClassificationForm.jsx

```diff
@@ -97,32 +97,30 @@ const ClassificationForm = ({ obj_id, taxonomyList }) => {
 
   // Custom form widget for the classifications to format and display the contexts as well
   const CustomClassificationWidget = ({ value, onChange, options }) => {
```

Comment:
@guynir42 I think same thing about the return statement for here and the CustomProbabilityWidget as well. I think that's perhaps what's causing the `test_classifications` test to fail

Reply:
Yep, this still needs to be fixed.